src/reference/00-Getting-Started/09-Multi-Project.md

```diff
@@ -106,9 +109,9 @@ you can control aggregation per-task. For example, to avoid aggregating
 the `update` task:
 
 ```scala
-lazy val root = (project in file(".")).
-  aggregate(util, core).
-  settings(
+lazy val root = (project in file("."))
+  .aggregate(util, core)
+  .settings(
     aggregate in update := false
   )
 
```

Comment:
Below there is an example

`lazy val core = project.dependsOn(util)`

As all the other examples here added `in file(".")`, should we be consistent in that example?